packages/gestalt/src/Callout.js

```diff
@@ -32,7 +33,7 @@ type Props = {|
    * Adds a dismiss button to Callout. See the [Dismissible variant](https://gestalt.pinterest.systems/web/callout#Dismissible) for more info.
    * The `accessibilityLabel` should follow the [Accessibility guidelines](https://gestalt.pinterest.systems/web/callout#Accessibility).
    */
-  dismissButton?: {| accessibilityLabel: string, onDismiss: () => void |},
+  dismissButton?: {| accessibilityLabel: ?string, onDismiss: () => void |},
```

Comment:
I think here should be

`{| accessibilityLabel?: string, onDismiss: () => void |},`

Reply:
DONE